### ISSUE TITLE
merge.exe crashes with error, "IndexError: list index out of range"

### ISSUE THREAD

**Issue by GoogleCodeExporter** [SEBDYA/googletransitdatafeed#153]
```
What steps will reproduce the problem?
1. run merge.exe

What is the expected output? What do you see instead?
Expect a merged file.  A crash report is seen instead, "IndexError: list 
index out of range"  Both feeds validate separately (with warnings) when 
running feedvalidator.exe.  This only happens with merge.exe.

What version of the product are you using? On what operating system?
1.2.  Windows XP Professional

Please provide any additional information below.
Traceback (most recent call last):
  File "merge.py", line 1734, in <module>
  File "inspect.pyc", line 888, in trace
  File "inspect.pyc", line 876, in getinnerframes
  File "inspect.pyc", line 840, in getframeinfo
  File "inspect.pyc", line 510, in findsource
IndexError: list index out of range

I can provide the zip files, if needed...each is 20MB.
```

Original issue reported on code.google.com by `devin.br...@gmail.com` on 16 Mar 2009 at 9:08

Attachments:
- [merge-crash.txt](https://storage.googleapis.com/google-code-attachments/googletransitdatafeed/issue-153/comment-0/merge-crash.txt)

**Comment by GoogleCodeExporter** [comment]
```
Update--merging with merge.exe version 1.1.9, I got an error: the dates are not 
disjoint. Found the error in calendar.txt, fixed it, and it merged properly in 
1.1.9, 
but 1.2.0 gave a new error, see attached.  The resulting merged file was 
successfully 
validated with version 1.2.0 of feedvalidator.exe.  
```

Original comment by `devin.br...@gmail.com` on 17 Mar 2009 at 10:42

Attachments:
- [merge-crash.txt](https://storage.googleapis.com/google-code-attachments/googletransitdatafeed/issue-153/comment-1/merge-crash.txt)

**Comment by GoogleCodeExporter** [comment]
```
First crash is because you didn't quote the filenames with a space... but it 
should 
crash. Fix pending in http://codereview.appspot.com/110058

Second crash is dup of issue 165, fixed in 1.2.1
```

Original comment by `tom.brow...@gmail.com` on 21 Aug 2009 at 8:45
- Changed state: **Accepted**
- Added labels: **App-Merge**, **Language-Python**, **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Fix for first crash submitted in r1088, to be in next release.
```

Original comment by `tom.brow...@gmail.com` on 21 Aug 2009 at 2:00
- Changed state: **Fixed**
